app/controllers/api/v1/users_controller.rb

```diff
@@ -5,4 +5,33 @@ def show
 
     render json: UserSerializer.new(user).serializable_hash, status: :ok
   end
+
+  def update
+    user = User.find(params[:id])
+    user.update!(user_params)
+
+    render json: UserSerializer.new(user).serializable_hash, status: :ok
+  end
+
+  def create
+    user = User.create!(user_create_params)
+
+    render json: UserSerializer.new(user).serializable_hash, status: :created
+  end
+
+  private
+
+  # TODO: Do we want to allow them to update the info that is collected from
+  #       Google Oauth?
+  #       :full_name, :email, :google_id, :google_image_url
```

Comment:
Do we want to allow them to update the info that is collected from Google Oauth?
`:full_name`, `:email`, `:google_id`, `:google_image_url`

Reply:
If we can get the `get_user` action from FE working smoothly and efficiently, definitely.  Right now, the only time these get updated is upon initial user registration (we just direct straight to dashboard if login is successful, with the way current session controller flow is built)